crates/sui/src/client_commands.rs

```diff
@@ -468,14 +468,17 @@ impl SuiClientCommands {
                 let sender = context.try_get_object_owner(&gas).await?;
                 let sender = sender.unwrap_or(context.active_address()?);
 
-                let build_config =
-                    resolve_lock_file_path(build_config, Some(package_path.clone()))?;
+                let config = resolve_lock_file_path(build_config, Some(package_path.clone()))?;
+                let run_bytecode_verifier = true;
+                let print_diags_to_stderr = true;
+
+                let sui_build_config = BuildConfig {
```

Comment:
I would say, just alias `config` that is what I've seen most commonly in our codebase (but no strong feelings).